README.md

```diff
@@ -84,18 +91,18 @@ Memory usage:   85.1M out of 985.4M
 
 ```
 $ multipass shell dancing-chipmunk
-Welcome to Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0-42-generic x86_64)
 
+Welcome to Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0-42-generic x86_64)
 ...
 ```
 
-Don't forget to logout (or Ctrl-D) or you may find yourself heading all the
-way down the Inception levels... ;)
+Don't forget to logout (or Ctrl-D) or you may find yourself heading all the way down Inception levels... ;)
```

Comment:
I know this PR did not change this, but should the `Inception` be non-capitalised?

Reply:
It's the name of a movie which is probably why it is capitalized.

Reply:
ok, the literal meaning of the word does not make sense here, only the movie insinuated does.

Reply:
I think capitalising Inception here makes sense, as it's a reference to the movie... "inception" on its own means beginning and doesn't convey the meaning of recursiveness or nested levels. I'll leave it as it is for now :)